src/events/create.js

```diff
@@ -6,8 +6,16 @@ exports.run = (client, logger, message) => {
 
     const waitTime = moment.duration(100, 'seconds').milliseconds();
     conversationHandler
-        .ask('Please enter a name for the event', waitTime)
-        .then('Please enter a start time for the event', waitTime)
+        .ask('Please enter a name for the event', waitTime, undefined, (content) => {
+            return `Please confirm that the event name "${content}" is corrrect.`;
+        })
+        .then('Please enter a start time for the event', waitTime, (content) => {
+            if (content === 'banana') {
```

Comment:
This is probably worth fixing to be real code, not that I don't appreciate banana time